Dependency_Injection/Configuration/DIConfigure.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Dependency_Injection.Repositories;
+using Dependency_Injection.Service;
+using Dependency_Injection.Service.Notification.Concrete;
+using Dependency_Injection.Services;
+
+namespace Dependency_Injection.Configuration
+{
+    public static class DIConfigure
+    {
+        public static IServiceCollection AddServices(this IServiceCollection services)
+        {
+            //Register services
+            services.AddScoped<IOrderRepository, OrderRepository>();
+            services.AddScoped<IOrderService, OrderService>();
+
+            //Register List
+            services.AddScoped<INotificationService, EmailNotificationService>();
```

Comment:
would you please give a quick explanation on why do we need the notification registered as scoped

Reply:
Because we need to create a different object (instance) on each user request